src/Okta.Sdk.IntegrationTests/PoliciesScenarios.cs

```diff
@@ -0,0 +1,1482 @@
+﻿// <copyright file="PoliciesScenarios.cs" company="Okta, Inc">
+// Copyright (c) 2014 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
```

Comment:
This file is pretty big, I wonder if it would be good to break it up a bit?  For example, you could break it up by policy type, or break out policy rule tests from policy tests.

Reply:
@nbarbettini what do you think? I am using the existent "style" here.